.github/workflows/publish-release.yml

```diff
@@ -0,0 +1,31 @@
+name: publish-release
+on:
+  push:
+    tags:
+      - "*.Final"
+jobs:
+  docker:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v2
+      - name: Docker meta
+        id: meta
+        uses: docker/metadata-action@v3
+        with:
+          images: jboss/wildfly
+          tags: type=ref,event=tag
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      - name: Login to DockerHub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
```

Comment:
could you please rename it to `${{ secrets.TOKEN }}`?

Reply:
done